src/sentry/api/base.py

```diff
@@ -40,7 +40,7 @@
 ONE_HOUR = ONE_MINUTE * 60
 ONE_DAY = ONE_HOUR * 24
 
-LINK_HEADER = '<{uri}&cursor={cursor}>; rel="{name}"; results="{has_results}"; cursor="{cursor}"'
+LINK_HEADER = u'<{uri}&cursor={cursor}>; rel="{name}"; results="{has_results}"; cursor="{cursor}"'
```

Comment:
Not particularly related.. but a good practice on format strings.

Reply:
Should we default to unicode strings everywhere instead?

Reply:
Hmm, what do you mean?

Reply:
It's probably a good idea to mark all strings as explicitly binary or unicode. Not sure if powertools can help us with that.

Reply:
Oh I see what you're talking about byk... It might be safer to mark them individually than blanket turn on unicode strings.